diff --git a/astropy/coordinates/tests/test_solar_system.py b/astropy/coordinates/tests/test_solar_system.py
index 06acc1756c..154497d0c4 100644
--- a/astropy/coordinates/tests/test_solar_system.py
+++ b/astropy/coordinates/tests/test_solar_system.py
@@ -1,4 +1,5 @@
 from urllib.error import HTTPError
+import sys

 import numpy as np
 import pytest
@@ -396,6 +397,7 @@ def test_ephemeris_wrong_input(ephemeris, expected_error):


 @pytest.mark.skipif(not HAS_JPLEPHEM, reason="requires jplephem")
+@pytest.mark.skipif(sys.version_info < (3, 12), reason="old jplehpem caused problems, skip this instead")
 def test_ephemeris_local_file_not_ephemeris():
     # NOTE: This test currently leaves the file open (ResourceWarning).
     # To fix this issue, an upstream fix is required in jplephem
